packages/wrangler/src/api/dev.ts

```diff
@@ -169,7 +169,7 @@ export async function unstable_dev(
 		compatibilityDate: options?.compatibilityDate,
 		compatibilityFlags: options?.compatibilityFlags,
 		ip: options?.ip,
-		inspectorPort: options?.inspectorPort,
+		inspectorPort: options?.inspectorPort ?? 0,
```

Comment:
would it make sense to have this fallback to the default inspector port of `9222`?

Reply:
No the CLI can makes those decisions for the default value but the default in the general case should be 0 (random port)